Crash with libGDX 1.9.9

When using the latest libGDX v1.9.9 and the latest libgdx-inGameConsole v1.0.0b-SNAPSHOT, if you create a GUIConsole with the default empty constructor, the app crashes with the following stacktrace:

```
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: java.lang.NoSuchMethodError: com.badlogic.gdx.InputMultiplexer.getProcessors()Lcom/badlogic/gdx/utils/Array;
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:133)
Caused by: java.lang.NoSuchMethodError: com.badlogic.gdx.InputMultiplexer.getProcessors()Lcom/badlogic/gdx/utils/Array;
	at com.strongjoshua.console.GUIConsole.hasStage(GUIConsole.java:259)
	at com.strongjoshua.console.GUIConsole.resetInputProcessing(GUIConsole.java:233)
	at com.strongjoshua.console.GUIConsole.<init>(GUIConsole.java:152)
	at com.strongjoshua.console.GUIConsole.<init>(GUIConsole.java:127)
	at com.strongjoshua.console.GUIConsole.<init>(GUIConsole.java:113)
	at com.strongjoshua.console.GUIConsole.<init>(GUIConsole.java:86)
	at com.strongjoshua.console.GUIConsole.<init>(GUIConsole.java:73)
```

It seems to be due to a breaking change introduced to InputMultiplexer in 1.9.9.

--- a/src/com/strongjoshua/console/GUIConsole.java
+++ b/src/com/strongjoshua/console/GUIConsole.java
@@ -256,7 +256,7 @@ public class GUIConsole extends AbstractConsole {
 			return processor == stage;
 		}
 		InputMultiplexer im = (InputMultiplexer)processor;
-		Array<InputProcessor> ips = im.getProcessors();
+		SnapshotArray<InputProcessor> ips = im.getProcessors();
 		for (InputProcessor ip : ips) {
 			if (hasStage(ip)) {
 				return true;

